floodviz/maputils.py

```diff
@@ -0,0 +1,58 @@
+import requests
+
+def site_dict(site_list, url_prefix):
+    """Puts site data into a dictionary
+
+    Args:
+        site_list: A list of site ids to be queried on NWIS
+        url_prefix: The beginning of the NWIS url
```

Comment:
You should also describe the type of url_prefix -- so something like "A string containing the NWIS site url".